clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/loadbalancer/UpsertGoogleSslLoadBalancerAtomicOperation.groovy

```diff
@@ -287,7 +287,7 @@ class UpsertGoogleSslLoadBalancerAtomicOperation extends UpsertGoogleLoadBalance
         loadBalancingScheme: 'EXTERNAL',
         IPProtocol: description.ipProtocol,
         IPAddress: description.ipAddress,
-        portRange: description.certificate ? "443" : description.portRange,
```

Comment:
I verified that the UI (as it exists now) sends `description.portRange == "443"`, rather than relying on this ternary, so this is OK.